### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by alexma1835**

### CODE REVIEW

**Comment by 1jackdev** [comment]
this looks like a mistake. maybe remove all the gibberish

**Comment by alexma1835** [comment]
> this looks like a mistake. maybe remove all the gibberish

> alright, will delete the codes

File: README.md [hunk 1]
```diff
@@ -4,3 +4,19 @@ Job Market Analysis 2024
 # Contributors
 Alex Ma
 Jack Mayer
+jdgkahfgakdjghalkhfgfghidfshgkhdskghf
```
Comment (1jackdev) [inline README.md:7]: whats up

Comment (alexma1835) [reply README.md:7]: whats uppppp

**Review by 1jackdev — APPROVED** [review]
looks good, lets merge

**Pull request merged**